backends/gaudi/tgi-entrypoint.sh

```diff
@@ -10,8 +10,8 @@ fi
 # Check if ATTENTION environment variable is set to paged
 if [[ "$ATTENTION" == "paged" ]]; then
   # Check if Llama-4 is in the command line arguments
-  if [[ "$*" == *"Llama-4"* ]]; then
-    echo 'ATTENTION=paged and Llama-4 detected'
+  if [[ "$*" == *"Llama-4"* || "$*" == *"Qwen3"* ]]; then
+    echo 'ATTENTION=paged and Llama-4 or Qwen3 detected'
     pip install git+https://github.com/huggingface/transformers.git@29338949
```

Comment:
Transformers v4.52 should be released today, let's wait for it and update this line?

Reply:
@yuanwu2017 We can use Transformers v4.52.2 now

Reply:
cannot merge. found a bug.

Reply:
Done.

Reply:
Sounds good